### PR TITLE
LPS-187651 Add link to Ai Creator documentation

### PR DESCRIPTION
**Pull request by Stephy6**
https://liferay.atlassian.net/browse/LPS-187651

### CODE REVIEW

File: modules/apps/ai-creator/ai-creator-openai-web/src/main/resources/META-INF/resources/ai_creator_modal/AICreatorModal.tsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ import {TextContent} from './TextContent';
 
 interface Props {
 	getCompletionURL: string;
+	learnResources: object;
```
Comment (sandrodw3) [inline modules/apps/ai-creator/ai-creator-openai-web/src/main/resources/META-INF/resources/ai_creator_modal/AICreatorModal.tsx:21]: Could we type this a bit more specifically? 😄

**Review by sandrodw3 — COMMENTED** [review]
Hey @Stephy6 this looks well! 😄 just suggested a tiny improvement!

File: modules/apps/ai-creator/ai-creator-openai-web/src/main/resources/META-INF/resources/ai_creator_modal/AICreatorModal.tsx (hunk 1 above)
Comment (Stephy6) [reply modules/apps/ai-creator/ai-creator-openai-web/src/main/resources/META-INF/resources/ai_creator_modal/AICreatorModal.tsx]: Sure

**Review by sandrodw3 — APPROVED** [review]

**Comment by sandrodw3** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**